mongodbatlas/clusters_test.go

```diff
@@ -55,7 +59,13 @@ func TestClusters_ListClusters(t *testing.T) {
 						"diskIOPS": 1320,
 						"encryptEBSVolume": false,
 						"instanceSizeName": "M40",
-						"regionName": "US_WEST_2"
+						"regionName": "US_WEST_2",
+						"autoScaling": {
+							"compute": {
+							  "maxInstanceSize": "M40",
+							  "minInstanceSize": "M10"
```

Comment:
It doesn't really make sense to have the max size match the instanceSizeName, if we want to make it a bit more a real world example max is usually a cluster size or two (on average) more than the instanceSizeName a user starts with and min one or two below - M60 would make more sense in this case.

Reply:
You're right, I'll change it to M60 in max since is the same and it should be higher than instanceSizeName like you said, thank you very much melissa